<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Soitin</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/soitin.png</normaloff>:/soitin.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="page" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="PlaylistView" name="treeView" >
          <property name="editTriggers" >
           <set>QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="showDropIndicator" stdset="0" >
           <bool>false</bool>
          </property>
          <property name="dragEnabled" >
           <bool>true</bool>
          </property>
          <property name="dragDropMode" >
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="verticalScrollMode" >
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <property name="sortingEnabled" >
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2" />
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;List</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionAddFiles" />
    <addaction name="actionAddFolder" />
    <addaction name="actionClear" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menu_Playback" >
    <property name="title" >
     <string>&amp;Playback</string>
    </property>
    <addaction name="actionPrevious" />
    <addaction name="actionPlay" />
    <addaction name="actionStop" />
    <addaction name="actionNext" />
    <addaction name="separator" />
    <addaction name="actionShuffle" />
    <addaction name="actionFullScreen" />
    <addaction name="separator" />
    <addaction name="actionShowPlaylist" />
    <addaction name="actionShowVideo" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Playback" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Tools</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious" />
   <addaction name="actionPlay" />
   <addaction name="actionStop" />
   <addaction name="actionNext" />
   <addaction name="separator" />
   <addaction name="actionShuffle" />
   <addaction name="actionFullScreen" />
   <addaction name="separator" />
   <addaction name="actionShowPlaylist" />
   <addaction name="actionShowVideo" />
  </widget>
  <action name="actionAddFiles" >
   <property name="text" >
    <string>&amp;Add Files...</string>
   </property>
  </action>
  <action name="actionAddFolder" >
   <property name="text" >
    <string>Add &amp;Folder...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionPrevious" >
   <property name="text" >
    <string>&amp;Previous</string>
   </property>
   <property name="shortcut" >
    <string>Z</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPlay" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>P&amp;lay</string>
   </property>
   <property name="shortcut" >
    <string>X</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionStop" >
   <property name="text" >
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut" >
    <string>C</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionNext" >
   <property name="text" >
    <string>&amp;Next</string>
   </property>
   <property name="shortcut" >
    <string>V</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionShuffle" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Sh&amp;uffle</string>
   </property>
   <property name="shortcut" >
    <string>S</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionShowPlaylist" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Pl&amp;aylist</string>
   </property>
  </action>
  <action name="actionShowVideo" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Video</string>
   </property>
  </action>
  <action name="actionFullScreen" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Full S&amp;creen</string>
   </property>
   <property name="shortcut" >
    <string>F</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionClear" >
   <property name="text" >
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>Save as...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaylistView</class>
   <extends>QTreeView</extends>
   <header>playlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
